### DISCUSSION TITLE
Limit honey to one country IP address

### DISCUSSION THREAD

**Discussion by y0d4a** [telekom-security/tpotce#1182] [Q&A]
Hi, is it possible to limit visibility of tpotce to only one country?

tnx.

**Comment by lcia-projects** [comment]
not easily.. you'd have to know all the ip ranges used for that country.. that can change some.. easier way would be to collect all the data then just filter for the country you want to look at in Kibana.

**Comment by Clankcoll** [comment]
Well i would say it depends on your setup.

You could use GEOIP blocking in your Firewall and only allow that one Country for your TPOT.

Or like @lcia-projects  wrote collect all data and then only filter for that one Country.
